File: src/examples/validateDocument/basic.cs
using Criipto.Signatures;
using Criipto.Signatures.Models;

using (var client = new CriiptoSignaturesClient("{YOUR_CRIIPTO_CLIENT_ID}", "{YOUR_CRIIPTO_CLIENT_SECRET}")) {
    var validateDocumentInput = new ValidateDocumentInput
    {
        pdf = new Byte[64], // Should be the bytes of a PDF file
    };

    // Create signature order
    var result = await client.ValidateDocument(validateDocumentInput);

    Console.WriteLine(result.valid); // Whether or not document is valid
    Console.WriteLine(result.errors); // Any errors
    Console.WriteLine(result.fixable); // If errors are fixable
}

File: src/examples/validateDocument/basic.py
from criipto_signatures import CriiptoSignaturesSDKSync
from criipto_signatures.models import ValidateDocumentInput


client = CriiptoSignaturesSDKSync(
  "{YOUR_CRIIPTO_CLIENT_ID}", "{YOUR_CRIIPTO_CLIENT_SECRET}"
)

result = client.validateDocument(
  ValidateDocumentInput(
    pdf=bytes("...", "utf-8")  # Should be a bytes object or base64 encoded string.
  )
)

print(result.valid)  # Whether or not document is valid
print(result.errors)  # Any errors
print(result.fixable)  # If errors are fixable
